edx_rest_api_client/auth.py

```diff
@@ -58,26 +58,38 @@ def __call__(self, r):
 
 
 class SuppliedJwtAuth(AuthBase):
-    """Attaches a supplied JWT to the given Request object."""
+    """
+    Attaches a supplied JWT to the given Request object.
+    """
```

Comment:
@nedbat: Are most of these changes just personal preference, or was this breaking a rule? Most of these seem fine according to https://peps.python.org/pep-0257/.

Reply:
A number of the docstrings used single-line but with extra spaces at the end, and some were multi-line.  I made them all consistent.